packages/cli/cloud/src/create-project/action.ts

```diff
@@ -0,0 +1,61 @@
+import inquirer from 'inquirer';
+import { AxiosError } from 'axios';
+import { defaults } from 'lodash/fp';
+import type { CLIContext, ProjectAnswers, ProjectInput } from '../types';
+import { tokenServiceFactory, cloudApiFactory, local } from '../services';
+
+function handleError(ctx: CLIContext, error: Error) {
+  const tokenService = tokenServiceFactory(ctx);
+  const { logger } = ctx;
+
+  logger.debug(JSON.stringify(error));
+  if (error instanceof AxiosError) {
+    switch (error.response?.status) {
```

Comment:
If it's 5XX should we handle it to show it's our problem not theres? at the moment it's not clear imo

Reply:
Yes, we handle 503 for now.
I think the default message doesn't include the user as part of the problem: "We encountered an issue while creating your project. Please try again in a moment."